docs/stdlib/generic.rst

```diff
@@ -321,12 +330,14 @@ Generic
 
     :index: find strpos strstr position array
 
-    A polymorphic function to find index of an element in a sequence.
+    Returns the index of an element in a ``haystack`` set by the ``needle``
+    value.
 
-    When the *haystack* is :eql:type:`str` or :eql:type:`bytes`,
-    return the index of the first occurrence of *needle* in it.
+    When ``haystack`` is a :eql:type:`str` or :eql:type:`bytes` type, this
+    will return the index of the first (as well as only) occurring ``needle``
+    in it.
 
-    When the *haystack* is an :eql:type:`array`, return the index of
+    When ``haystack`` is an :eql:type:`array`, return the index of
     the first occurrence of the specific *needle* element. For
```

Comment:
```suggestion
    the first occurrence of the element passed as *needle*. For
```

Reply:
We should also change line 346 for consistency. We have been using a future tense to talk about what functions will return given a parameter (at least, we have been using that tense fairly consistently outside the one-liners). That line could read 
```
If the *needle* is not found, this function will return ``-1``.
```